framework/tlvf/src/include/tlvf/BaseClass.h

```diff
@@ -42,7 +42,7 @@ class BaseClass {
     uint8_t *m_buff_ptr__;
     const size_t m_buff_len__;
     const bool m_parse__;
-    bool m_finalized;
+    bool m_finalized__;
```

Comment:
Why is this rename needed?

Reply:
To align with other variable names which are explicitly used in tlvf.py so we use the __ suffix to mark that.